dynamiqs/utils/tensor_types.py

```diff
@@ -32,17 +32,37 @@ def to_tensor(
     device: torch.device | None = None,
     is_complex: bool = False,
 ) -> Tensor:
-    """Convert an array-like object (or a list of array-like objects) to a tensor.
+    r"""Convert an array-like object (or a list of array-like objects) to a tensor.
 
     Args:
-        x: QuTiP quantum object or NumPy array or Python list or PyTorch tensor or
-            list of these types. If `None` or empty list, returns an empty tensor of
-            shape _(0)_.
+        x _(array-like or list of array-like)_: QuTiP quantum object or NumPy array or
+            Python list or PyTorch tensor or list of these types. If `None` or empty
+            list, returns an empty tensor of shape _(0)_.
         dtype: Data type of the returned tensor.
         device: Device on which the returned tensor is stored.
+        is_complex: Whether the returned tensor is complex-valued.
 
     Returns:
         Output tensor.
+
+    Examples:
+        >>> import numpy as np
+        >>> import qutip as qt
+        >>> dq.to_tensor(qt.fock(3, 1))
```

Comment:
I'm wondering. Should we use the full module name in our examples? e.g. `dynamiqs.to_tensor(qt.fock(3,1))` here. Since we never define the import